src/Form/CodeMirror.scss

```diff
@@ -0,0 +1,8 @@
+.codemirror-form {
```

Comment:
wäre es nicht besser in einen "hangar" namensraum zu geben? würde ich grundsätzlich machen damit man nicht versehentlich wenn man unsere komponenten einbindet etwas überschreibt was den selben namen hat.

Reply:
gefixt